nss-cohort-40/bangazon-ecommerce-api-upper-class-twits-1
# Tb customer model

# Description
Created Customer model in models directory

## Type of change


# Testing Instructions
Check the customer.py file in the models directory. Ensure it has all properties from ERD. Ensure it is imported into the models directory __init__.py

# Checklist:
